storybookjs/storybook
# Addon-info - Fix immutable props issues with React 16

Fix storybooks/storybook#1890 - issue with react fiber and immutable props.

Issue: See discussion here #1890 

## What I did
I cloned the object before passing it in as props to a component (since we used to attempt to mutate the props, causing issues with react fiber)

## How to test
When using storybook with react 16 and addon-info, the Source section of the info would throw the `TypeError: "paddingLeft" is read-only` error. This PR fixes that issue.

Is this testable with jest or storyshots?
No?

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
